3.Tests/UI/StepDefinitions/CommonStepDefinition.cs

```diff
@@ -1,12 +1,29 @@
-﻿using TechTalk.SpecFlow;
-using Todoly.Core.UIElements.Drivers;
+﻿using System;
+using TechTalk.SpecFlow;
 using Todoly.Views.WebAppPages;
 
 namespace Todoly.Tests.UI.Steps.Commons;
 
 public class CommonSteps
 {
     private readonly ScenarioContext _scenarioContext;
+    private string? _currentView;
+    public string CurrentView
+    {
+        get => _currentView!;
+        set
+        {
+            if (!string.IsNullOrWhiteSpace(value))
+            {
+                _currentView = value;
+            }
+
+            if (string.IsNullOrWhiteSpace(_currentView))
```

Comment:
This if statement should be on the getter

Reply:
Fixed